build.sbt

```diff
@@ -57,24 +57,39 @@ lazy val commonSettings = releaseSettings ++ Seq(
 
 lazy val cFlags = Seq(
   scalacOptions ++= (CrossVersion.partialVersion(scalaVersion.value) match {
-    case Some((2, n)) if n <= 10 =>
+
+
+
+    case Some((2, n)) if n == 11 =>
       Seq(
         "-language:existentials",
         "-language:higherKinds",
-        "-language:implicitConversions"
+        "-language:implicitConversions",
+        "-Ypartial-unification"
       )
-    case Some((2, n)) if n == 11 =>
+    case Some((2, n)) if n == 12 =>
       Seq(
         "-language:existentials",
         "-language:higherKinds",
         "-language:implicitConversions",
+        "-Xfuture",
```

Comment:
It looks like we are removing certain compiler options from our 2.12 build (e.g. `-Ywarn-extra-implicit`, which appears in `ScalacOptions.All` but not in this list, will no longer be used in 2.12).

I noticed Fabian's pull request omitted the same flags. Am I reading it incorrectly?

Reply:
I fixed it. The intent was to remove options from the 2.13 build, because some options are now deprecated.